css/at-rules/keyframes.json

```diff
@@ -0,0 +1,134 @@
+{
+  "css": {
+    "at-rules": {
+      "@keyframes": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@keyframes",
+          "support": {
+            "webview_android": {
+              "prefix": "-webkit-",
+              "version_added": true
+            },
+            "chrome": [
+              {
+                "version_added": "43"
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": true
+              }
+            ],
+            "chrome_android": {
+              "version_added": null
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": [
+              {
+                "version_added": "16"
+              },
+              {
```

Comment:
We need to add two {} blocks to describe the story of -webkit- prefixes in Firefox (added in 44 behind a pref, without a pref in 49)

{
  "version_added": "49",
  "prefix": "-webkit-"
},
{
  "version_added": "44",
  "prefix": "-webkit-",
  "flag": {
    "type": "preference",
    "name": "layout.css.prefixes.webkit",
    "value_to_set": "true"
  }
},